network-api/networkapi/donate_banner/models.py

```diff
@@ -51,11 +51,42 @@ class DonateBanner(TranslatableMixin, PreviewableMixin, models.Model):
             "or a valid query string starting with ? (Ex: ?form=donate)."
         ),
     )
-    background_image = models.ForeignKey(
+    foreground_image = models.ForeignKey(
         "wagtailimages.Image",
         models.PROTECT,
         related_name="+",
     )
+    bg_image = models.ForeignKey(
+        "wagtailimages.Image",
+        models.PROTECT,
+        related_name="+",
+        null=True,
+        blank=True,
+    )
+
+    TAILWIND_COLORS = [
+        ("bg-gray-60", "Gray"),
```

Comment:
Would it be possible to include `tw-` in these class names? It would make the classes easier to search for in the future!